form/2d/p5js/closethetriangle/index.md

```diff
@@ -0,0 +1,9 @@
+---  
+layout: p5js
+author: [Jakob von Kietzell]
+author-url: [https://incom.org/profil/7939]
+title: [closethetriangle]
+tags: [P5.js,[FORM], [2D], Law of Closure]
+permalink: /[FORM]/[closethetriangle]/
```

Comment:
no `[]` here. form is written lowercase.  See the example https://raw.githubusercontent.com/fabianmoronzirfas/gestalten-in-code/master/grid/p5js/lines/index.md